locales/en/public.json

```diff
@@ -4,31 +4,31 @@
   },
   "AboutDescription": {
     "BUGS": "Bugs",
-    "FILE_A_REPORT": "File a Report",
+    "FILE_A_REPORT": "File a report",
     "HOMEPAGE": "Homepage",
-    "KNOWN_ISSUES": "Known Issues",
+    "KNOWN_ISSUES": "Known issues",
     "LICENSE": "License",
-    "MAILING_LIST": "Mailing List",
-    "OPEN_SOURCE_LICENSE": "Open Source License",
-    "VERSION": "version"
+    "MAILING_LIST": "Mailing list",
+    "OPEN_SOURCE_LICENSE": "Open source license",
+    "VERSION": "Version"
   },
   "AppLayout": {
     "APP_LAUNCHER": {
       "ABOUT": "About",
       "DOCUMENTATION": "Documentation",
       "GUIDED_TOUR": "Guided tour",
       "HELP": "Help",
-      "QUICKSTARTS": "Quick Starts"
+      "QUICKSTARTS": "Quick starts"
     }
   },
   "AutomatedAnalysisCard": {
     "CARD_DESCRIPTION": "Assess common application performance and configuration issues.",
     "CARD_DESCRIPTION_FULL": "Creates a recording and periodically evaluates various common problems in application configuration and performance. Results are displayed with scores from 0-100 with colour coding and in groups. This card should be unique on a dashboard.",
-    "CARD_TITLE": "Automated Analysis",
+    "CARD_TITLE": "Automated analysis",
     "CRITICAL_RESULTS_one": "{{count}} Critical Result",
     "CRITICAL_RESULTS_other": "{{count}} Critical Results",
     "ERROR_TEXT": "Cryostat was unable to generate an automated analysis report.",
-    "ERROR_TITLE": "Automated Analysis Error",
+    "ERROR_TITLE": "Automated analysis Error",
```

Comment:
Which rule applies here to make it `AaE` instead of `AAE` or `Aae`?

Reply:
The `Automated` is capitalized( `A`) because it is the beginning of the sentence.

Reply:
Looking at this [https://github.com/cryostatio/cryostat-web/pull/1260#discussion_r1611668081](https://github.com/cryostatio/cryostat-web/pull/1260#discussion_r1611668081)
The only reason `Automated` is capitalized is because it is at the beginning of the sentence. should we go back to `AAE`?